packages/vue/src/components/atoms/SfButton/SfButton.stories.js

```diff
@@ -155,6 +166,12 @@ Disabled.args = {
   ...Primary.args,
 };
 
+export const Submit = Template.bind({});
+Submit.args = {
+  type: "submit",
+  ...Primary.args,
+};
```

Comment:
@JohnAntonios  What do you think about adding `content: "Submit"` here?

Reply:
Thanks @AdamPawlinski!

That's a great suggestion, it would make it more clear to the viewer! I have updated it. (Also had to add the `content` prop to the `prop` table for it to work).

Reply:
You don't need to add `content` to the prop table. We don't want it as it is just added for users' convenience. We use it to easily change the button's text.

Reply:
You just need to add it below `...Primary.args` to overwrite the value added in Primary story.

Reply:
Ah I see now, thanks @AdamPawlinski ! I have made the change.